diff --git a/node_modules/@types/react-native/globals.d.ts b/node_modules/@types/react-native/globals.d.ts
index 7f27293..7f50705 100644
--- a/node_modules/@types/react-native/globals.d.ts
+++ b/node_modules/@types/react-native/globals.d.ts
@@ -232,6 +232,7 @@ interface XMLHttpRequestEventTarget {
     //  removeEventListener(type: string, listener: EventListenerOrEventListenerObject): void;
 }
 
+//@ts-ignore
 interface XMLHttpRequestUpload extends EventTarget, XMLHttpRequestEventTarget {
     addEventListener<K extends keyof XMLHttpRequestEventTargetEventMap>(
         type: K,
